### DISCUSSION TITLE
Can Select.Option not require children?

### DISCUSSION THREAD

**Discussion by jiahao-c** [arco-design/arco-design#418] [General]
Currently, we have to do this in Arco:
```jsx
<Select.Option value="a"> a </Select.Option>
```

It would be nice if we can also do this in Arco:
```jsx
<Select.Option value="a" />
```

The above syntax is possible in both Semi and Antd.

**Comment by PengJiyuan** [comment]
@MisterLuffy Maybe we can treat value as children, if there are no children?
